### ISSUE TITLE
Incompatibility with projects using pydantic 1.x

### ISSUE THREAD

**Issue by fontclos** [dribia/dbt2pdf#1]
When adding `dbt2pdf` to a project that uses an "old" version of pydantic (< 2.0), poetry cannot resolve dependencies.

```bash
$ poetry add dbt2pdf
Using version ^0.0.1 for dbt2pdf

Updating dependencies
Resolving dependencies... (1.8s)

    Because no versions of dbt2pdf match >0.0.1,<0.0.2
 and dbt2pdf (0.0.1) depends on pydantic (>=2.8.2,<3.0.0), dbt2pdf (>=0.0.1,<0.0.2) requires pydantic (>=2.8.2,<3.0.0).
    Because metricflow (0.206.0) depends on pydantic (>=1.10.0,<1.11.0)
 and no versions of metricflow match >0.206.0,<0.207.0, metricflow (>=0.206.0,<0.207.0) requires pydantic (>=1.10.0,<1.11.0).
(1) Thus, dbt2pdf (>=0.0.1,<0.0.2) is incompatible with metricflow (>=0.206.0,<0.207.0).

```

**Comment by airibarne** [comment]
This should be fixed in [v0.0.4](https://github.com/dribia/dbt2pdf/releases/tag/v0.0.4). @fontclos, please check and close if everything works as expected.

**Comment by fontclos** [comment]
Works like a charm 🎉
